vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "crons": [
+    {
+      "path": "/api/cron",
+      "schedule": "0 0 * * 1"
```

Comment:
@noelfernandez98 
This might be interesting for you. 
https://vercel.com/docs/cron-jobs#cron-expressions

This hits the `/api/cron` endpoint, every Monday at 12am.